security/cmd/node_agent_k8s/main.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 Istio Authors
```

Comment:
we have this already https://github.com/istio/istio/tree/master/security/cmd/node_agent_k8s/nodeagent/main.go

Reply:
And we will merge this dir node_agent_k8s in to node_agent/ soon